### PR TITLE
<FilePicker /> - add unidriver

### PR DESCRIPTION
**Pull request by justenor**

### CODE REVIEW

**Comment by mosheKerbel** [comment]
u have a conflict in spec

File: src/FilePicker/FilePicker.uni.driver.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { baseUniDriverFactory, ReactBase } from '../../test/utils/unidriver';
```
Comment (mosheKerbel) [inline src/FilePicker/FilePicker.uni.driver.js]: ReactBase is not being used

**Comment by mosheKerbel** [comment]
u have a conflict in FilterPicker.spec

File: src/FilePicker/FilePicker.uni.driver.js (hunk 1 above)
Comment (justenor) [reply src/FilePicker/FilePicker.uni.driver.js]: Fixed

**Comment by mosheKerbel** [comment]
2 failing tests of unrelated async timeout
merged

**Pull request merged**